src/app/docs/README.md

```diff
@@ -29,6 +29,10 @@ Framework including the file you are reading right now.
 This directory contains all of the cluster implementations included in the CHIP
 ZCL Application Framework.
 
+### <code>/src/app/zap></code>
```

Comment:
`### <code>/src/app/zap></code>` -> `### <code>/src/app/zap-templates</code>`

If the intent of this folder is to only contains templates, then I would prefer `zap-templates` since `zap` really sounds like the software itself.

And don't forget to fix the extra `>` :)